pkg/tcpip/network/fragmentation/fragmentation.go

```diff
@@ -140,3 +155,28 @@ func (f *Fragmentation) release(r *reassembler) {
 		f.size = 0
 	}
 }
+
+// TimeOut function generates ICMP TTL Error message (Fragment reassembly time exceeded message).
+func (f *Fragmentation) TimeOut(netHeader buffer.View, vv buffer.VectorisedView) {
+	vv = vv.Clone(nil)
+	hdr := buffer.NewPrependable(int(R.MaxHeaderLength()) + header.ICMPv4MinimumSize + header.IPv4MinimumSize)
```

Comment:
What is R?

Reply:
Earlier, we thought of passing 'r' as parameter to Process. But this was resulting in failure of some test cases in fragmentation_test.go
Also, it involved additional effort to fix those test cases. So, we decided to introduce a new variable 'R'.

Reply:
Please no globals. It has to be a parameter.